AutoMaxLair.py

```diff
@@ -498,12 +475,13 @@ def select_pokemon(inst) -> str:
     take_pokemon = False  # Set to True if a non-legendary shiny is found.
     reset_game = False  # Set to True in some cases in non-default modes.
 
-    if inst.num_caught == 4 and (CHECK_ATTACK_STAT is True or CHECK_SPEED_STAT is True):
+    if inst.num_caught == 4 and (inst.check_attack_stat is True or inst.check_speed_stat is True):
```

Comment:
I think you can omit `is True`—inst.check_attack_stat is already `True` so the expression is basically `True is True` which evaluates to `True` (redundant).

Reply:
haha you're welcome ! Just did the required changes ^^